site/content/blog/2021-03-01-whats-new-in-svelte-march-2021.md

```diff
@@ -0,0 +1,105 @@
+---
+title: What's new in Svelte: March 2021
+description: Call for Svelte Summit Speakers + Better SSR, Web Component and non-HTML5 compilation support.
```

Comment:
I think ESLint TypeScript support is a big one for quite a few people so I suggest to add that to the headline
```suggestion
description: Call for Svelte Summit Speakers + Better SSR, non-HTML5 compilation option and TypeScript support for ESLint.
```

Reply:
This headline is a bit confusing to me because it's unclear if better SSR support was added or if we have a "Call for Better SSR" especially since it uses a `+` where the others are separated with `,`

I might reword as:
```
description: Call for Svelte Summit Speakers. Improved SSR, non-HTML5 compilation targets, and ESLint TypeScript support.
```